Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -201,10 +204,11 @@ public function showAction(string $node): void
         }
 
         $contentRepository = $this->contentRepositoryRegistry->get($nodeAddress->contentRepositoryId);
-        $subgraph = $contentRepository->getContentGraph($nodeAddress->workspaceName)->getSubgraph(
+        $uncachedSubgraph = $contentRepository->getContentGraph($nodeAddress->workspaceName)->getSubgraph(
             $nodeAddress->dimensionSpacePoint,
             VisibilityConstraints::frontend()
         );
+        $subgraph = new ContentSubgraphWithRuntimeCaches($uncachedSubgraph, $this->subgraphCachePool);
```

Comment:
we probably need an api way of doing this as thats sometimes the case on the boundary where we dont have a $node yet and cannot use the cr registry